source/components/HexMetrics.js

```diff
@@ -102,11 +102,13 @@ export default class HexMetrics extends React.Component {
       const warn = (count.idealNHex === 0 && count.nHex === 0) ?
         <i className='fa fa-exclamation-triangle' /> :
         null
+      let className = count.deviation === 0 ? 'fade' : ''
+      if (count.disable) { className += ' disabled' }
```

Comment:
Can we get some linebreaks in that block? Should probably set up a linter rule for it…